### ISSUE TITLE
Zenroom's npm package is published with incompatible version of core-js

### ISSUE THREAD

**Issue by transducer** [DECODEproject/Zenroom#106]
The [`package.json` in the javascript bindings of GitHub](https://github.com/DECODEproject/Zenroom/blob/master/bindings/javascript/package.json#L49) mentions as dependencies:

```
  "dependencies": {
    "@babel/node": "^7.6.3",
    "core-js": "^3.1.4",
    "regenerator-runtime": "*"
  },
```

the [published package on `npm`](https://unpkg.com/zenroom@1.1.0/package.json) has:

```
  "dependencies": {
    "core-js": "^3.6.5"
  }
```

when doing `npm install Zenroom` and running it with the 3.6.5 version of core-js from shadow-cljs (a cljs build tool) I see this error:

```
The required JS dependency "path" is not available, it was required by "node_modules/zenroom/dist/lib/zenroom.js".

Searched in: (...)/node_modules

You probably need to run:
  npm install path
```

when running `npm install core-js@3.1.4 regenerator-runtime` (the dependencies mentioned on GitHub) this error disappears and Zenroom works. 

So the working dependencies seem to be the ones mentioned on GitHub, and not those actually published with the package.

**Comment by puria** [comment]
Hi @transducer thanks for reporting I made a prelease you can install it by `npm i zenroom@next` it should address your problem. Can you please confirm? If that works I'll make a release.

**Comment by transducer** [comment]
Thanks. This works.

**Comment by puria** [comment]
cool released 1.1.1
